model/student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom, age)
+    super(age)
+    @classroom = classroom
+  end
```

Comment:
- [ ] You need to inherit all parameters from the person class and initialize them, so please fix that.
[hint] it will be like that:
```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
end
```